### DISCUSSION TITLE
Can I embed a section of one page inside another page, in a quarto website?

### DISCUSSION THREAD

**Discussion by emmansh** [quarto-dev/quarto-cli#3409] [Q&A]
Having separate pages in my website, I'd like to embed a section from page X inside page y. Can this be achieved with quarto's native features?

In his [answer](https://github.com/quarto-dev/quarto-cli/discussions/2900#discussioncomment-4167789), @cscheid provided two options for hyperlinking between separate website pages:

> Use one of:
> 
> * relative paths: `[link to first blog post](../blog_post_1/index.qmd)`. Relative paths are always resolved with respect to the document they reside in.
> * "absolute" paths: `[link to first blog post](/blog_post_1/index.qmd)`. Absolute paths are always resolved with respect to the project root directory.

I wonder whether we could take this mechanism one step forward and embed page sections in other pages. I've tried something along the lines of:
```markdown
<iframe src = root/posts/blog_post_1/index.qmd#my-section></iframe>
<!-- or with double quotes? -->
<iframe src = "root/posts/blog_post_1/index.qmd#my-section"></iframe>
```
But it doesn't work. 

Does Quarto have a trick up its sleeve for that?

**Comment by dragonstyle** [comment]
I'm not aware of way to embed a section like this, but you could literally place the html in both places using `{{< include >}}` - not sure if this is what you're looking for (it obviously makes it more of a chore to create shared content).

https://quarto.org/docs/authoring/includes.html

**Reply by cscheid** [reply]
I will just warn here that relative paths with includes are _quite unintuitive_. The includes are handled _completely textually_, and so they will be resolved relative to the path of the document that is calling the include, and not the path of the include file itself.

In general, if you're using includes that have paths, I recommend using project-relative paths (ie paths starting with `/`) in the includes to minimize the risks for confusion.

**Reply by emmansh** [reply]
Thank you both. I've opened #3421 since  suggested `{{< include >}}` throws an error.

**Reply by cscheid** [reply]
`#code-example` is not part of the filename; remove that part.

**Reply by emmansh** [reply]
I see, thanks. Circling back to the original question then -- given that I can't specify `#code-example` means that I also can't "include" a specific section?

**Reply by dragonstyle** [reply]
Yes that is correct- you would need to make that section a separate file and include it in both other files instead.
